include/deal.II/hp/fe_collection.h

```diff
@@ -174,6 +174,12 @@ namespace hp
     unsigned int
     n_blocks() const;
 
+    /**
+     * Return the maximal degree over all elements of this collection.
```

Comment:
Maybe say something like "Return the maximum of the values returned by FiniteElement::get_degree() over all ..."